back-office/shop-parameters/traffic-and-seo/seo-and-urls-page.md

```diff
@@ -0,0 +1,22 @@
+# **SPECIFICATIONS - SEO & URLS PAGE**
+
+
+## Grid
+
+_MetaGridDefinitionFactory.php_ Default action in the button: Edit Other available actions in the menu: Delete Row action: Edit
+
+Row action does not apply to the ID's checkbox column.
+
+
+## URLs set up
+
+[TO BE COMPLETED]
+
+**Force update of friendly URL.** At first, this option was only available for products, in the Shop Parameters > Product Settings page but it has been decided to make it available for all URLs. So it is now a SEO & URLs parameter to be found in this dedicated section of the back office, cf. _to do_ issue #[21386](https://github.com/PrestaShop/PrestaShop/issues/21386).
+
+A friendly URL is generated from the page/item name. By default, it is disabled. Enabling this option will **automatically regenerate it every time the merchant modifies it and saves a new name**. As a consequence, it automatically updates the friendly URL column in the 'SEO & URLs' listing of the page.
```

Comment:
If I understand well, the friendly URL field is automatically updated according to the name ?
If yes, maybe we should specify to what elements it's applied: products, categories, CMS pages, SEO & urls pages, and maybe others ?
Currently friendly URLs for CMS page and categories (and maybe others, I didn't check all) are automatically updated when the name is changed. While the existing parameter is disabled by default for products.
So, I am wondering if it's a good idea to disable this new parameter by default
I'm afraid some merchants who are used to having the friendly updated automatically for categories or CMS pages might be surprised